dask/blockwise.py

```diff
@@ -662,12 +724,24 @@ def make_blockwise_graph(func, output, out_indices, *arrind_pairs, **kwargs):
     deserializing = kwargs.pop("deserializing", False)
     func_future_args = kwargs.pop("func_future_args", None)
     return_key_deps = kwargs.pop("return_key_deps", False)
+    io_deps = kwargs.pop("io_deps", {})
```

Comment:
All these lines look like an obsolete hack to make Python 2 happy (you could not put named keyword-only arguments after *args). Could you clean up and put the actual arguments in the function signature?

Reply:
Yeah, exactly - Sounds good.